pixi.toml

```
@@ -141,17 +141,21 @@ test-benchmark = 'pytest datashader/tests --benchmark --gpu'
 # =================== DOCS ====================
 # =============================================
 [feature.doc.dependencies]
-nbsite = ">=0.8.4,<0.9.0"
+nbsite =  ">=0.9.0a2"
 numpydoc = "*"
 sphinxcontrib-mermaid = "*"
 sphinx-reredirects = "*"
 
 [feature.doc.tasks]
 _docs-generate-rst = 'nbsite generate-rst --org holoviz --project-name datashader'
 _docs-generate = 'nbsite build --what=html --output=builtdocs --org holoviz --project-name datashader'
+_docs-install = 'python -m pip install --no-deps --disable-pip-version-check -e .'
 
 [feature.doc.tasks.docs-build]
-depends-on = ['_docs-generate-rst', '_docs-generate']
+depends-on = ['_docs-install', '_docs-generate-rst', '_docs-generate']
```

```suggestion
depends-on = ['_docs-generate-rst', '_docs-generate']

[feature.doc.tasks.docs-build-full]
depends-on = ['_docs-install', 'download-data', '_docs-generate-rst', '_docs-generate']
```

I would still want it to be able to run the docs without installing.